easybuild/easyconfigs/t/Tk/Tk-8.6.4-iomkl-2016.07.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = 'ConfigureMake'
+
+name = 'Tk'
+version = '8.6.4'
+
+homepage = 'http://www.tcl.tk/'
+description = """Tk is an open source, cross-platform widget toolchain that provides a library of basic elements for
+ building a graphical user interface (GUI) in many different programming languages."""
+
+toolchain = {'name': 'iomkl', 'version': '2016.07'}
+
+source_urls = ["http://prdownloads.sourceforge.net/tcl"]
+sources = ['%(namelower)s%(version)s-src.tar.gz']
+
+patches = ['Tk-8.6.4_different-prefix-with-tcl.patch']
```

Comment:
Sure. I've only changed it in the files I'm adding - it's like this in several others too. Would you like me to change it in them all?